crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -1107,8 +1136,20 @@ impl Protobuf for Ics20Withdrawal {
             .transpose()
             .map_err(Ics20WithdrawalError::invalid_bridge_address)?;
 
+        if bridge_address.is_some() {
```

Comment:
I am pretty unhappy that we now have a JSON formatted payload in here. I wonder if we can make a breaking change to these transactions and add another variant to the protobuf that makes this a protobuf.

Reply:
Considering this would be network breaking, I'm not sure what we can really do about it now, barring a hard fork :/

We could make a new action which uses a protobuf-formatted memo instead and follow the same upgrade path as the new `ValidatorUpdateV2` action, but I'm not sure how worth it that would be.